src/main/webapp/app/shared/participant-scores/participant-scores.component.ts

```diff
@@ -0,0 +1,21 @@
+import { Component, Input } from '@angular/core';
+import { ParticipantScoreDTO } from 'app/shared/participant-scores/participant-scores.service';
+
+@Component({
+    selector: 'jhi-participant-scores',
+    templateUrl: './participant-scores.component.html',
+    styles: [],
+})
+export class ParticipantScoresComponent {
+    @Input()
+    participantScores: ParticipantScoreDTO[] = [];
+    @Input()
+    isLoading = false;
+    extractParticipantName = (participantScoreDTO: ParticipantScoreDTO) => {
+        if (participantScoreDTO.userName) {
+            return `${participantScoreDTO.userName}`;
+        } else {
+            return `${participantScoreDTO.teamName}`;
```

Comment:
What if there is no teamName?

Reply:
This can never happen. Either a result is generated from a student or from a team